tests/test_xsd.py

```diff
@@ -162,16 +194,20 @@ def test_extra_testcase_property_mk8s(testdir, properly_decorated_test_function)
     # Add another property element for the testcase.
     xml_doc.find('./testcase/properties').append(etree.Element('property',
                                                                attrib={'name': 'extra', 'value': 'fail'}))
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('mk8s')))
```

Comment:
Since an external program is accessing this method it should be marked as public.